### PR TITLE
Task/add storybook

### PR DESCRIPTION
**Pull request by solomontang**

### CODE REVIEW

File: src/utils/Logger.js [hunk 1]
```diff
@@ -40,7 +38,8 @@ function isNonEmptyString(value :any) {
 
 function getMessagePrefix(loggerLevel :string, loggerName :string) {
 
-  return `[${format(new Date(), TIMESTAMP_FORMAT)} ${loggerLevel.toUpperCase()} ${__PACKAGE__}] ${loggerName}`;
+  // eslint-disable-next-line max-len
+  return `[${DateTime.local().toSQL({ includeOffset: false })} ${loggerLevel.toUpperCase()} ${__PACKAGE__}] ${loggerName}`;
```
Comment (UnsungHero97) [inline src/utils/Logger.js]: In all other `Logger` files, I got rid of any date/format dependencies, and just changed it to this:

```
return `[${(new Date()).toISOString()} ${loggerLevel.toUpperCase()} ${__PACKAGE__}] ${loggerName}`;
```

You can just copy it directly: https://github.com/openlattice/lattice-js/blob/develop/src/utils/Logger.js

File: package.json [hunk 2]
```diff
@@ -27,65 +27,73 @@
...
+    "eslint-import-resolver-webpack": "~0.11.0",
+    "eslint-loader": "~2.1.0",
+    "eslint-plugin-flowtype": "~3.10.0",
+    "eslint-plugin-import": "~2.17.0",
+    "eslint-plugin-jest": "~22.6.0",
+    "eslint-plugin-jsx-a11y": "~6.2.0",
+    "eslint-plugin-react": "~7.13.0",
+    "file-loader": "~4.0.0",
+    "flow-bin": "~0.102.0",
     "html-webpack-plugin": "^3.2.0",
```
Comment (UnsungHero97) [inline package.json]: `~`

File: package.json [hunk 3]
```diff
@@ -27,65 +27,73 @@
...
-    "immutable": "^4.0.0-rc.9",
-    "jest": "^23.5.0",
-    "lattice": "^0.42.0",
-    "lattice-auth": "^0.8.0",
+    "jest": "~24.8.0",
+    "jest-styled-components": "~6.3.3",
     "npm-run-all": "^4.1.0",
```
Comment (UnsungHero97) [inline package.json]: `~`

File: package.json [hunk 4]
```diff
@@ -27,65 +27,73 @@
...
-    "react": "~16.4.0",
-    "react-dom": "~16.4.0",
+    "react": "~16.8.0",
+    "react-dom": "~16.8.0",
     "rimraf": "^2.6.0",
```
Comment (UnsungHero97) [inline package.json]: `~`

File: package.json [hunk 5]
```diff
@@ -27,65 +27,73 @@
...
-    "stylelint-config-standard": "^18.2.0",
-    "stylelint-config-styled-components": "^0.1.1",
-    "stylelint-processor-styled-components": "^1.3.0",
+    "style-loader": "~0.23.1",
+    "styled-components": "~4.3.0",
+    "stylelint": "~10.1.0",
+    "stylelint-config-standard": "~18.3.0",
+    "stylelint-config-styled-components": "~0.1.1",
+    "stylelint-processor-styled-components": "~1.8.0",
     "uuid": "^3.3.0",
```
Comment (UnsungHero97) [inline package.json]: `~`

File: package.json [hunk 6]
```diff
@@ -27,65 +27,73 @@
...
-    "webpack-cli": "^3.1.0",
-    "webpack-dev-server": "^3.1.0"
+    "webpack": "~4.33.0",
+    "webpack-cli": "~3.3.0",
+    "webpack-dev-server": "~3.6.0"
   },
   "peerDependencies": {
-    "@babel/polyfill": "^7.0.0",
+    "@babel/polyfill": "~7.4.0",
     "immutable": "^4.0.0-rc.9",
```
Comment (UnsungHero97) [inline package.json]: Exact version, please.

File: package.json [hunk 7]
```diff
@@ -27,65 +27,73 @@
...
+    "@storybook/addon-options": "~5.1.0",
+    "@storybook/addon-storysource": "~5.1.0",
+    "@storybook/addons": "~5.1.0",
+    "@storybook/react": "~5.1.0",
+    "@storybook/theming": "~5.1.0",
+    "babel-eslint": "~10.0.0",
+    "babel-jest": "~24.8.0",
+    "babel-loader": "~8.0.0",
+    "babel-plugin-styled-components": "~1.10.0",
+    "css-loader": "^3.0.0",
```
Comment (UnsungHero97) [inline package.json]: `~`

File: package.json [hunk 8]
```diff
@@ -102,6 +110,8 @@
     "postversion": "git push && git push --tags",
     "test": "jest --config config/jest/jest.config.js",
     "test:travis": "jest --maxWorkers=4 --config config/jest/jest.config.js",
-    "test:watch": "jest --config config/jest/jest.config.js --watch --coverage=false"
+    "test:watch": "jest --config config/jest/jest.config.js --watch --coverage=false",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```
Comment (UnsungHero97) [inline package.json]: Please alphabetize. Also, we're using `:` as a convention, so this would be `build:storybook`.

Comment (solomontang) [reply package.json]: `build-storybook` was created by the install process
```npx -p @storybook/cli sb init```
I'm not really sure what it does yet.

**Pull request merged**